deploy/examples/cluster-test.yaml

```diff
@@ -26,6 +26,11 @@ metadata:
   name: my-cluster
   namespace: rook-ceph # namespace:cluster
 spec:
+  network:
+    provider: multus
+    selectors:
+      public: rook-ceph/public-net
+      cluster: rook-ceph/cluster-net
```

Comment:
Ignore changes to `cluster-test.yaml`. This is for testing and will be removed before final review.